ndmg/stats/qa_skullstrip.py

```diff
@@ -0,0 +1,272 @@
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+# Created by Vikram Chandrashekhar.
+# Edited by Greg Kiar, Eric Bridgeford and Chuankai Luo.
+# Email: Greg Kiar @ gkiar@jhu.edu   cluo16@jhu.edu
+
+import warnings
+
+warnings.simplefilter("ignore")
+import os
+import re
+import sys
+import numpy as np
+import nibabel as nb
+# import ndmg.utils as mgu
+from argparse import ArgumentParser
+from scipy import ndimage
+from matplotlib.colors import LinearSegmentedColormap
+import matplotlib as mpl
+
+mpl.use("Agg")  # very important above pyplot import
+from nilearn.plotting.edge_detect import _edge_map as edge_map
+import matplotlib.pyplot as plt
+
+
+
+
+def gen_overlay_pngs(
+    brain, origional, outdir, loc=0, mean=False, minthr=2, maxthr=95, edge=False):
+    """
+    outdir: directory where output png file is saved
+    fname: name of output file WITHOUT FULL PATH. Path provided in outdir.
+    """
+    try:
+        origional_name = get_filename(origional)
+    except:
+        origional_name = 'compare_origional_data_and_brain'
+    brain_data = nb.load(brain).get_data()
+    if brain_data.ndim == 4:  # 4d data, so we need to reduce a dimension
+        if mean:
+            brain_data2 = brain_data.mean(axis=3)
+        else:
+            brain_data2 = brain_data[:, :, :, loc]
+    else:  # dim=3
+        brain_data2 = brain_data
+
+    cmap1 = LinearSegmentedColormap.from_list("mycmap1", ["white", "magenta"])
+    cmap2 = LinearSegmentedColormap.from_list("mycmap2", ["white", "green"])
+
+
+    fig = plot_overlays(brain_data2, origional, [cmap1, cmap2], minthr, maxthr, edge)
+
+    # name and save the file
+    fname = "qa_skullstrip__" + origional_name + ".png"
+    fig.savefig(outdir + "/" + fname, format="png")
+    # plt.close()
+
+
+def opaque_colorscale(basemap, reference, vmin=None, vmax=None, alpha=1):
+    """
+    A function to return a colorscale, with opacities
+    dependent on reference intensities.
+    **Positional Arguments:**
+        - basemap:
+            - the colormap to use for this colorscale.
+        - reference:
+            - the reference matrix.
+    """
+    reference = reference
+    if vmin is not None:
+        reference[reference > vmax] = vmax
+    if vmax is not None:
+        reference[reference < vmin] = vmin
+    cmap = basemap(reference)
+    maxval = np.nanmax(reference)
+    # all values beteween 0 opacity and 1
+    opaque_scale = alpha * reference / float(maxval)
+    # remaps intensities
+    cmap[:, :, 3] = opaque_scale
+    return cmap
+
+
+
+
+def plot_overlays(b0, origional, cmaps=None, minthr=2, maxthr=95, edge=False):
```

Comment:
You and other members of your team have a function named `plot_overlays` that does different things. Please name this function something else

Reply:
fixed it in latest version

Reply:
> fixed it in latest version
resolve comments yourself whenever you have resolved something

Reply:
![image](https://user-images.githubusercontent.com/55203069/70834530-96452500-1dc8-11ea-8729-175f4a2aa504.png)

Reply:
![image](https://user-images.githubusercontent.com/55203069/70834610-c8568700-1dc8-11ea-84b0-31dbf468c8c0.png)